book/src/examples/intro.md

```diff
@@ -0,0 +1,12 @@
+# Examples
+
+This chapter contains various examples that demonstrates how hax can
+be used to prove properties about programs. Each example is
+self-contained. hax being a tool that can extract Rust to various
+backends, this section provides examples for each backends..
```

Comment:
"each backend" (without s), and probably only one dot at the end.

Reply:
Good catch, thanks!